examples/README.md

```diff
@@ -0,0 +1,33 @@
+## Requirements
```

Comment:
The directory name seems to imply multiple examples, but there is only one. You might want to future proof by nesting under `examples/simple` or something of the sort.